https://issues.redhat.com/browse/ACM-13088 (#6770)
open-cluster-management/rhacm-docs: governance/gatekeeper_operator/general_support.adoc

[#general-support]
= General support

> To understand the support you receive from the Gatekeeper operator, see the following list:

Reviewer: ```suggestion
To understand the support you receive from the {gate}, see the following list:
```

> - The most current version of the {gate} is fully supported, along with the preceding version, and all z-stream releases of those versions.
> - The preceding version from the most current version receives maintenance support and releasing relevant security vulnerability fixes. 
> - All {ocp} versions under standard support are supported with the {gate}.
> - The {gate} is not supported on {ocp-short} versions that are under extended support or have reached their end-of-life status.

Reviewer: Based on the [style guide for lists](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#definition-lists), I suggest the following changes. I would also recommend that we remove the last bullet point since this is a list about what's supported. Since ocp versions that receive standard support is mentioned we might not need to call out what's not supported. The bullet point seems inconsistent with the list. Alternatively, we can keep the statement and add it as a note:
```suggestion
- Supports current version of the {gate}, preceding versions, and all z-stream releases of those versions.
- Receive maintenance support and relevant security vulnerability fixes for preceding and current versions. 
- Support for all {ocp} versions that receive standard support.
- The {gate} is not supported on {ocp-short} versions that are under extended support or have reached their end-of-life status.
```

OR

```suggestion
- Supports current version of the {gate}, preceding versions, and all z-stream releases of those versions.
- Receive maintenance support and relevant security vulnerability fixes for preceding and current versions. 
- Support for all {ocp} versions that receive standard support.
*Note*: The {gate} is not supported on end-of-life {ocp-short} versions or versions that receive extended support.
```
